### PR TITLE
Update final interaction states snapshots - RSC-1534

### PR DESCRIPTION
**Pull request by EmilyBermudez88**
https://issues.sonatype.org/browse/RSC-1534

Confirming all snapshots are updated & old ones are removed before the final merge.

### CODE REVIEW

**Comment by entroform** [comment]
Oo I believe there are some colour variables that are also no longer in use like `--nx-color-interactive-shadow-focus`
I think we should deprecate them here before we merge

File: gallery/src/styles/CssVariables/CssVariablesPage.tsx [hunk 1]
```diff
@@ -205,15 +205,21 @@ const CssVariablesPage = () => {
...
+            </NxWarningAlert>
           </PropertyDocItem>
           <PropertyDocItem propertyVar="--nx-box-shadow-dropdown">
             The standard box-shadow to apply to dropdown elements.
           </PropertyDocItem>
           <PropertyDocItem propertyVar="--nx-drop-shadow-focus">
-            Again, the standard shadow to apply to focused elements in addition to a border or outline. On elements
-            where <NxCode>box-shadow</NxCode> isn't supported, such as SVG elements, use this instead
-            as the value of a <NxCode>filter</NxCode> <NxCode>drop-shadow</NxCode>.
+            <NxWarningAlert>
```
Comment (entroform) [inline gallery/src/styles/CssVariables/CssVariablesPage.tsx:217]: This is just a small thing but I wonder  what you think about just having the deprecate message inside the alert and the rest outside.

Comment (EmilyBermudez88) [reply gallery/src/styles/CssVariables/CssVariablesPage.tsx:217]: > This is just a small thing but I wonder what you think about just having the deprecate message inside the alert and the rest outside.

I agree with you. However, the other deprecated examples had both the 'deprecated' reasoning & definition inside the alert. I'm happy to change all of them for consistency sake - what do you think?

Comment (entroform) [reply gallery/src/styles/CssVariables/CssVariablesPage.tsx:217]: Ah okay, I guess we can keep it the same with other ones for consistency sake 👍

**Review by avnvbhatta — APPROVED** [review]
🚀

**Pull request merged**